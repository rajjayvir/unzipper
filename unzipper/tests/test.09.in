0x1f
0x6e 0x40 0x00
0x63 0x41 0x00
0x76 0x62 0x00
0x68 0x62 0x40
0x75 0x52 0x80
0x6f 0x43 0x00
0x72 0x44 0x00
0x6c 0x45 0x00
0x0a 0x36 0x00
0x6b 0x88 0x00
0x6a 0x88 0x10
0x31 0xa8 0x20
0x39 0xb8 0x24
0x33 0xc8 0x26
0x38 0xc8 0x27
0x7a 0xa8 0x28
0x78 0xa8 0x2c
0x77 0x98 0x30
0x71 0x98 0x38
0x66 0x68 0x40
0x62 0x58 0x80
0x74 0x49 0x00
0x73 0x5a 0x00
0x6d 0x5a 0x80
0x69 0x4b 0x00
0x64 0x5c 0x00
0x70 0x7c 0x80
0x79 0x7c 0xa0
0x67 0x6c 0xc0
0x65 0x4d 0x00
0x61 0x3e 0x00
0x00 0x00 0x00 0x00 
