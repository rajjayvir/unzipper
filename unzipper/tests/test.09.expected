0x6e : 4 0000
0x63 : 4 0001
0x76 : 6 001000
0x68 : 6 001001
0x75 : 5 00101
0x6f : 4 0011
0x72 : 4 0100
0x6c : 4 0101
0x0a : 3 011
0x6b : 8 10000000
0x6a : 8 10000001
0x31 : 10 1000001000
0x39 : 11 10000010010
0x33 : 12 100000100110
0x38 : 12 100000100111
0x7a : 10 1000001010
0x78 : 10 1000001011
0x77 : 9 100000110
0x71 : 9 100000111
0x66 : 6 100001
0x62 : 5 10001
0x74 : 4 1001
0x73 : 5 10100
0x6d : 5 10101
0x69 : 4 1011
0x64 : 5 11000
0x70 : 7 1100100
0x79 : 7 1100101
0x67 : 6 110011
0x65 : 4 1101
0x61 : 3 111
